src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -355,6 +363,13 @@ private void HandleRegularCommand(StartPageListItem item)
                     dvm.ShowNewFunctionDialogCommand.Execute(null);
                     break;
 
+                case ButtonNames.ShowGalleryUI:
+                    GalleryViewModel galleryViewModel = new GalleryViewModel(this.DynamoViewModel);
+                    GalleryView galleryView = new GalleryView(galleryViewModel);
+                    if (dvm.ShowGalleryUICommand.CanExecute(galleryViewModel))
+                        dvm.ShowGalleryUICommand.Execute(galleryView);
```

Comment:
Do not do this, we don't need `DynamoViewModel.ShowGalleryUICommand` command at all, we could keep the `GalleryView` as a data member of `StartPageView` class:

```csharp

GalleryView galleryView;

private void HandleRegularCommand(StartPageListItem item)
{
    switch (...)
    {
        case ButtonNames.ShowGalleryUI:
            ShowGalleryView();
            break;
    }
}

private void ShowGalleryView()
{
    if (galleryView == null) // On-demand creation.
    {
        var galleryViewModel = new GalleryViewModel(dynamoViewModel);
        galleryView = new GalleryView(galleryViewModel);
    }
}
```